src/Chapter2.hs

```diff
@@ -336,7 +336,11 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList _ ____ [] = []
```

Comment:
That's a very long underscore! 😅 
Any particular reason for writing it this way? 🙂

Reply:
Oh this is quite funny. Initially I had `subList start end [] = []` and then I got the warning that the variables were unused - must have accidently added too many underscores when swapping out the variable! 1 is more elegant agreed.